packages/rancher-pushprox/charts/templates/pushprox-clients.yaml

```diff
@@ -114,9 +115,12 @@ spec:
           value: /etc/ssl/push-proxy/push-proxy-key.pem
         - name: CACERT_FILE_TARGET
           value: /etc/ssl/push-proxy/push-proxy-ca-cert.pem
+        securityContext:
+          runAsNonRoot: false
```

Comment:
Running as root is required in order to be able to copy files within the hostPath.